src/test/java/org/cactoos/text/ReplacedTextTest.java

```diff
@@ -0,0 +1,24 @@
+package org.cactoos.text;
+
+import org.cactoos.TextHasString;
+import org.hamcrest.MatcherAssert;
+import org.junit.Test;
+
+/**
+ * Test case for {@link ReplacedText}.
+ * @author Mehmet Yildirim (memoyil@gmail.com)
+ * @version $Id$
+ * @since 0.1
+ * @checkstyle JavadocMethodCheck (500 lines)
+ */
+public final class ReplacedTextTest {
+
+    @Test
+    public void replaceText() {
```

Comment:
Then we should have here a test case that verifies that if you have multiple occurences of the substring in your original string, only the first one gets replaced.